lib/vanagon/platform.rb

```diff
@@ -435,6 +435,29 @@ def add_build_repository(*args)
       end
     end
 
+    # Save the generic compiled archive and relevant metadata as packaging
+    # output. This will include a json file will all of the components/versions
```

Comment:
I think you meant the second 'will' to be 'with'

Reply:
woops, yup. fixing